bots/behaviorLib.lua

```diff
@@ -909,6 +909,23 @@ function behaviorLib.MoveExecute(botBrain, vecDesiredPosition)
 	local vecMyPosition = unitSelf:GetPosition()
 	local vecMovePosition = vecDesiredPosition
 	
+	if unitSelf:GetTypeName() ~= "Hero_Fairy" then
+		local teamBotBrain = core.teamBotBrain
+		local tTeleportData = teamBotBrain.tNymphoraTPData
+
+		local nCurrentTime = HoN.GetGameTime()
+		if tTeleportData.nChannelEndTime > nCurrentTime then

```

Comment:
This needs to check if tTeleportData is ~= nil first - otherwise all bots would error if Nymphora isn't in the match